src/pages/about.js

```diff
@@ -1,85 +1,101 @@
 import React from "react";
 import PropTypes from "prop-types";
+import styled from "styled-components";
 import Page from "../partials/page";
 import Showcase from "../partials/about/showcase";
 import { withRouteData } from "react-static";
 
-class About extends React.Component {
-  render() {
-    return (
-      <Page location={this.props.location}>
-        <h1> Victory: Charting for React and React Native</h1>
-        <p>
-          {/*
-           * TODO: Customize these buttons
-           * https://github.com/FormidableLabs/formidable-landers/issues/175
-           */}
-          <iframe
-            src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=star&count=true&size=large"
-            frameBorder="0"
-            scrolling="0"
-            width="160px"
-            height="30px"
-          >
-            &nbsp;
-          </iframe>
-          <iframe
-            src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=watch&count=true&size=large&v=2"
-            frameBorder="0"
-            scrolling="0"
-            width="160px"
-            height="30px"
-          >
-            &nbsp;
-          </iframe>
-          <iframe
-            src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=victory&type=fork&count=true&size=large"
-            frameBorder="0"
-            scrolling="0"
-            width="158px"
-            height="30px"
-          >
-            &nbsp;
-          </iframe>
-        </p>
-        <p>
-          Victory is a set of modular charting components for React and React
-          Native. Victory makes it easy to get started without sacrificing
-          flexibility. Create one of a kind data visualizations with fully
-          customizable styles and behaviors. Victory uses the same API for web
-          and React Native applications for easy cross-platform charting.
-        </p>
-        <p>
-          Victory is helmed by Formidable’s{" "}
-          <a href="https://github.com/boygirl"> Lauren Eastridge</a>.
-        </p>
-        <p>
-          <a href="https://github.com/FormidableLabs/victory/graphs/contributors">
-            See Victory Contributors
-          </a>
-        </p>
+const Divider = styled.hr`
+  border: none;
+  border-bottom: 1px solid ${({ theme }) => theme.color.red};
+  margin-top: 2rem;
+`;
 
-        <h2 id="showcase">Victory in Use</h2>
+const Section = styled.div`
+  margin-bottom: ${({ theme }) => theme.spacing.lg};
```

Comment:
I know we went back and forth about using the existing theme vs hardcoding and updating, I think for sanity's sake using the actual values instead of the closest theme value will bode well for us - then we can take those and make new theme spacing and clean it up. 

Same thing for any colors that aren't quite what the theme says, just hard coded it for now. I added a task in notion to update the colors - doing a search and find after that will be easy enough to replace the hard coded values with theme keys.